node-zerox/src/utils.ts

```diff
@@ -11,9 +11,13 @@ export const encodeImageToBase64 = async (imagePath: string) => {
 
 // Strip out the ```markdown wrapper
 export const formatMarkdown = (text: string) => {
-  const formattedMarkdown = text
-    .replace(/^```[a-z]*\n([\s\S]*?)\n```$/g, "$1")
-    .replace(/^```\n([\s\S]*?)\n```$/g, "$1");
+  let formattedMarkdown = text;
+  const regex = /```(?:\w+)?\n([\s\S]*?)\n```/g;
+
+  while (regex.test(formattedMarkdown)) {
```

Comment:
Maybe if this while loop is needed, we can also add a count and limit it to like 3 (just in case, so it does not go into an infinite loop). Probably unlikely that it is going to be wrapped with markdown 3 times haha